FraudProtection/build.gradle

```diff
@@ -9,8 +9,6 @@ android {
     defaultConfig {
         minSdkVersion modules.androidMinSdkVersion
         targetSdkVersion modules.androidTargetVersion
-        versionCode modules.sdkVersionCode
-        versionName modules.sdkVersionName
```

Comment:
Removed these properties form library modules since they no longer apply ([reference](https://developer.android.com/build/releases/past-releases/agp-4-1-0-release-notes)).